Gemfile.lock

```diff
@@ -0,0 +1,26 @@
+GEM
```

Comment:
Do we want to just ignore the lockfile for our use case?

Reply:
I thought about this a little more and I'm going to ignore it. While I would _always_ include this for a Rails app, I wouldn't in a Gem which is probably more similar to what we are doing. Our contributors likely use wildly different ruby version so locked versions will be extra annoying.

While commiting it does speed up bundle install by avoiding dependency resolution, we have a single gem in your Gemfile for now so the speed difference should be minimal.